pkg/controller/instance/instance_controller.go

```diff
@@ -171,12 +192,17 @@ func (r *ReconcileInstance) Reconcile(request reconcile.Request) (reconcile.Resu
 
 // reconcileTerminatingInstance triggers a helm uninstall for the created release
 func (r *ReconcileInstance) reconcileTerminatingInstance(ctx context.Context, log logr.Logger, harbor *registriesv1alpha1.Instance) error {
+	if harbor == nil {
+		return errors.New("no harbor instance provided")
```

Comment:
you can prevent the import of the additional `errors` package by using `fmt.Errorf()`:

```suggestion
		return fmt.Errorf("no harbor instance provided")
```

Reply:
We would still use the `error` package indirectly, because `fmt.Errorf` itself uses it, so there would be no benefit. I like to only use `fmt.Errorf` when really using format specifier verbs.